src/internal/operators/pluck.ts

```diff
@@ -51,7 +51,7 @@ export function pluck<T, R>(...properties: Array<string | number | symbol>): Ope
   return map((x) => {
     let currentProp: any = x;
     for (let i = 0; i < length; i++) {
-      const p = currentProp[properties[i]];
+      const p = currentProp?.[properties[i]] ?? undefined;
```

Comment:
I don't think the nullish coalescing operator (`??`) should be used here. It will convert values that are explicitly `null` to `undefined`. I think the optional chaining operator (`?.`) is sufficient.

Reply:
Right, made the change.

Reply:
It does mean that if the value is `null`, the operator will emit `null` instead of `undefined`, but I'm okay with that. We'll see what the others think. LGTM.